content/ember/v3/send-action.md

```diff
@@ -0,0 +1,87 @@
+---
+id: ember-component.send-action
+title: Use closure actions instead of `sendAction`
+until: '4.0.0'
+since: '3.3'
```

Comment:
The `since` here would be at least 3.4 (emberjs/ember.js#master represents 3.4 ATM)